Lab 6.ipynb

```diff
@@ -0,0 +1,395 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The 1 sigma uncertainty is from 190 to 210.\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "0.6826894921370859"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(1, 1)\n",
+    "\n",
+    "obs = 300\n",
+    "v_slice = Psd[obs]\n",
+    "v_slice = v_slice / np.sum(v_slice)\n",
+    "lin = np.arange(0, strength - 1, 1)\n",
+    "ax.step(lin, v_slice, where = 'mid')\n",
+    "ax.set_title(\"Vertical Slice at 300 Observed\")\n",
+    "ax.set_xlabel(\"True Signal\")\n",
+    "ax.set_ylabel(\"Probability\")\n",
+    "plt.show()\n",
+    "\n",
+    "prob1sigma = stats.norm.cdf(1, loc = 0, scale = 1)\n",
+    "\n",
+    "prob = 0\n",
+    "upper_index = 0\n",
+    "lower_index = 0\n",
+    "lower = False\n",
+    "for i in range(len(v_slice)):\n",
+    "    prob = prob + v_slice[i]\n",
+    "    if prob >= 1 - prob1sigma and lower == False:\n",
+    "        lower_index = i\n",
+    "        lower = True\n",
+    "    if prob >= prob1sigma:\n",
+    "        upper_index = i\n",
+    "        break\n",
+    "        \n",
+    "\n",
+    "print(\"The 1 sigma uncertainty is from \" + str(lower_index) + \" to \" + str(upper_index) + \".\")\n",
```

Comment:
Displaying the result as "mean ± uncertainty" would be more standard and would show symmetry